### ISSUE TITLE
bug: `import opentrons` is noisy

### ISSUE THREAD

**Issue by SyntaxColoring** [Opentrons/opentrons#5406]
# Overview

`import opentrons` prints some noisy and meaningless-to-users stuff.

This shows up:
* In Jupyter.
* With `opentrons_simulate`.
* With standalone scripts using `opentrons.execute.get_protocol_api()` or `opentrons_simulate.get_protocol_api()`.

This is minor, but it has worried customers before. They've thought that the "not found, loading defaults" messages meant that something was wrong.

It could also interfere with using the API as a library in some other script, which might need to tightly control its output.

# Current behavior

```shell
% python -c 'import opentrons'
Loading json containers...
Json container file load complete, listing database
Found 0 containers to add. Starting migration...
Database migration complete!
/Users/maxpm/.opentrons/deck_calibration.json not found. Loading defaults
/Users/maxpm/.opentrons/robot_settings.json not found. Loading defaults
%
```

The two "not found" messages at the bottom are printed to stderr.  The others are printed to stdout.

# Expected behavior

```shell
% python -c 'import opentrons'
%
```

**Comment by sfoster1** [comment]
This actually will be fixed in 3.18 by https://github.com/Opentrons/opentrons/pull/5413 . Because we now load the robot lazily, the work that produces the messages won't happen until the robot itself is imported.

That said, the messages will still happen when the robot is imported.

**Comment by SyntaxColoring** [comment]
It sounds like a fix for this has landed for APIv2, but the problem remains for APIv1. I don't think this is worth additional work to fix just for APIv1, so I'm retitling, tagging, and closing this ticket accordingly.
